defaults.yaml

```diff
@@ -65,8 +65,8 @@ net:
   master_ipv6_part: "c40a::"
   # base IPv6 address suffix for the minion nodes.
   # must be a string with 4 hex digits followed by "::"
-  # minion_ipv6_part: "c41e:"
-  minion_ipv6_part: "0001::"
+  minion_ipv6_part: "c41e::"
+  #minion_ipv6_part: "0001::"
```

Comment:
Why some of these are commented out?

Reply:
I found them already there! I will remove them

Reply:
Oh, I didn't know. Thanks!